### ISSUE TITLE
Wrong expression when 0 minutes is set as a number

### ISSUE THREAD

**Issue by santiagoalmeidabolannos** [Ataww/cronix#32]
using this data 
```
const data = {
    hour: 7,
    minute: '0',
    second: 0,
    dayOfMonth: '?',
    dayOfWeek: 'MON, WED, FRI'
}
```
the generated expression is `0 0 7 ? * MON,WED,FRI *` which is interpreted as `At 07:00:00am, on every Monday, Wednesday and Friday, every month` (expected result) but if we change the minute from `string` to a `number` the generated expression is different, something like `0 * 7 ? * MON,WED,FRI *` `Every minute at second :00 between 07am and 08am, on every Monday, Wednesday and Friday, every month`

**Comment by Ataww** [comment]
Hi,

Sorry I haven't given this project much attention for a while. I'll look into it next week.

**Comment by Ataww** [comment]
@santiagoalmeidabolannos This should be fixed with version 0.1.3
